Makefile

```
@@ -276,13 +285,15 @@ endif
 endif
 ifdef TRAVIS_TAG
 	@docker tag "$(DOCKER_REPOSITORY)" "$(DOCKER_REPOSITORY):$(VERSION_BASE)"
+	@docker tag "$(DOCKER_REPOSITORY)" "$(PCT_DOCKER_REPOSITORY):$(VERSION_BASE)"
```

I would add a condition on `PCT_DOCKER_REPOSITORY`
```suggestion
ifdef PCT_DOCKER_REPOSITORY
	@docker tag "$(DOCKER_REPOSITORY)" "$(PCT_DOCKER_REPOSITORY):$(VERSION_BASE)"
endif
```